tajo-storage/tajo-storage-hdfs/src/main/java/org/apache/tajo/storage/FileTablespace.java

```diff
@@ -343,7 +343,7 @@ public boolean accept(Path path) {
 
       FileStatus[] matches = fs.globStatus(p, inputFilter);
       if (matches == null) {
-        errors.add(new IOException("Input path does not exist: " + p));
+        LOG.warn("Input path does not exist: " + p);
       } else if (matches.length == 0) {
         errors.add(new IOException("Input Pattern " + p + " matches 0 files"));
```

Comment:
Would you share any reason?
I think that users can easily recognize this case even though it isn't regarded as invalid.